monitoring/prometheus/prometheus_staging.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval: 30s
+  external_labels:
+    monitor: 'evalai-monitor'
+
+scrape_configs:
+  - job_name: 'prometheus' 
+    static_configs: 
+      - targets: ['localhost:9090']
+
+  - job_name: 'django-prometheus'
+    metrics_path: '/prometheus/metrics'
+    static_configs:
+      - targets: ['https://staging.eval.ai']
```

Comment:
@burnerlee can you share why this will work? nginx will only redirect traffic from `https://staging.eval.ai/api` to django server. Have you tested this locally by setting up a nginx server with same redirection config?

Reply:
oh I see.. yeah it won't work.. should i add `/metrics` along with `/api` to redirect it to the django container? I'll test it on local as well this time